test/algo/compression/v2_5/utils/test_ds_transfotmers_evaluator.py

```diff
@@ -0,0 +1,189 @@
+from pathlib import Path
```

Comment:
will this file run in UT or IT?

Reply:
in UT

Reply:
which function in this file will be called in UT?

Reply:
I'm sorry, I took a closer look, and this is a unit test (UT). We don't have a separate UT testing folder, so it's placed here.